examples/packetcapture-dump/client/packetcapture-client.go

```diff
@@ -20,17 +20,23 @@ package main
 import (
 	"flag"
 	"fmt"
-	"net"
 	"os"
 	"time"
+
+	"github.com/google/gopacket/layers"
```

Comment:
import order

Reply:
done in https://github.com/netobserv/netobserv-ebpf-agent/pull/291/commits/beef4dd954e19bba5aec090685e889b2ee2e233f